src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -308,6 +309,15 @@ public class JdbcSinkConnectorConfig {
             .withDescription("A comma-separated list of regular expressions matching fully-qualified names of fields that "
                     + "should be excluded from change events. The field names must be delimited by the format <topic>:<field> ");
 
+    public static final Field USE_REDUCTION_BUFFER_FIELD = Field.create(USE_REDUCTION_BUFFER)
+            .withDisplayName("Controls whether to use reduction buffer by the connector to reduce the SQL load when duplicates are found")
+            .withType(Type.BOOLEAN)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 2))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.MEDIUM)
+            .withDefault(false)
+            .withDescription("Whether to use reduced buffer or not.");
```

Comment:
```suggestion
            .withDescription("A reduction buffer consolidates the execution of SQL statements by primary key to reduce the SQL load on the target database. When set to false (the default), each incoming event is applied as a logical SQL change. When set to true, incoming events that refer to the same row will be reduced to a single logical change based on the most recent row state.");
```
We generally prefer to have the display name be short while the description is considerably more verbose.

Reply:
thanks for review, doing changes